app/models/concerns/user_tracker.rb

```diff
@@ -0,0 +1,13 @@
+module UserTracker
```

Comment:
Is this needed now?

Reply:
good question. probably not =D let me check.

Reply:
specs pass without it, so we probably don't need it =)

Reply:
removed